recipes-graphics/wayland/weston-init/weston.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Weston Wayland Compositor
+RequiresMountsFor=/run
+Requires=weston.path
```

Comment:
ConditionPathExists lets the unit be skipped if it evaluates to false, but systemd doesn't start it later automatically if the condition suddenly becomes true.